legal-api/src/legal_api/services/document_record.py

```diff
@@ -0,0 +1,125 @@
+        validation_error = DocumentRecordService.validate_pdf(file, request.content_length)
+        if validation_error:
+            return {
+                'error': validation_error
+            }
+
+        file_content = file.read()
+
+        try:
+             # Read and encode the file content as base64
+            file_content = file.read()
+            file_base64 = base64.b64encode(file_content).decode('utf-8')
+
+            response_body = requests.post(
+                url,
+                params=query_params,
+                json={
+                    'filename': file.filename,
+                    'content': file_base64,
+                    'content_type': file.content_type,
+                },
+                headers={
+                    'x-apikey': current_app.config.get('DRS_X_API_KEY', ''),
+                    'Account-Id': current_app.config.get('DRS_ACCOUNT_ID', ''),
+                    'Content-Type': 'application/pdf'
+                }
+            ).json()
+
+            current_app.logger.debug(f'Upload file to document record service {response_body}')
+            return {
+                'documentServiceId': response_body['documentServiceId'],
+                'consumerDocumentId': response_body['consumerDocumentId'],
+                'consumerFilename': response_body['consumerFilename']
+            }
+        except Exception as e:
+            current_app.logger.debug(f"Error on uploading document {e}")
+            return {}
+
+    @staticmethod
+    def delete_document(document_service_id: str) -> dict:
+        """Delete document from Document Record Service."""
+        DRS_BASE_URL = current_app.config.get('DRS_BASE_URL', '') # pylint: disable=invalid-name
+        url = f'{DRS_BASE_URL}documents/{document_service_id}'
+
+        try:
+            response = requests.patch(
+                url, json={ 'removed': True },
+                headers={
+                    'x-apikey': current_app.config.get('DRS_X_API_KEY', ''),
+                    'Account-Id': current_app.config.get('DRS_ACCOUNT_ID', ''),
+                }
+            ).json()
+            current_app.logger.debug(f'Delete document from document record service {response}')
+            return response
+        except Exception as e:
+            current_app.logger.debug(f'Error on deleting document {e}')
+            return {}
+
+    @staticmethod
+    def validate_pdf(file, content_length) -> Optional[list]:
+        """Validate the PDF file."""
+        msg = []
+        try:
+            pdf_reader = PyPDF2.PdfFileReader(file)
+
+            # Check that all pages in the pdf are letter size and able to be processed.
+            if any(x.mediaBox.getWidth() != 612 or x.mediaBox.getHeight() != 792 for x in pdf_reader.pages):
+                msg.append({'error': _('Document must be set to fit onto 8.5” x 11” letter-size paper.'),
+                            'path': file.filename})
```

Comment:
This should be skipped for continuation in files. Ref: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/common_validations.py#L179

3rd argument is false in below lines
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/continuation_in.py#L130
https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/services/filings/validations/continuation_in.py#L160